### PR TITLE
feat: implement BYTE opcode

### LINKED ISSUE #25: Opcode: BYTE

### PR DESCRIPTION
**Pull request by ricomateo**
Closes #25 
This PR implements the code generation for the [BYTE](https://www.evm.codes/#1a?fork=cancun) opcode.

### CODE REVIEW

File: src/opcodes.rs [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#[derive(Debug)]
```
Comment (MegaRedHand) [inline src/opcodes.rs:1]: We can remove this file.

File: src/program.rs [hunk 2]
```diff
@@ -27,7 +27,7 @@ pub enum Opcode {
     // OR = 0x17,
     // XOR = 0x18,
     // NOT = 0x19,
-    // BYTE = 0x1A,
+    BYTE = 0x1A,
```
Comment (MegaRedHand) [inline src/program.rs:30]: Please change the opcode's status in the readme

File: tests/operations.rs [hunk 3]
```diff
@@ -140,6 +140,41 @@ fn pop_with_stack_underflow() {
     run_program_assert_revert(program);
 }
 
+#[test]
+fn push_push_byte() {
+    let mut value: [u8; 32] = [0; 32];
+    let desired_byte = 0xff;
+    value[16] = desired_byte;
+    let value: BigUint = BigUint::from_bytes_be(&value);
+    let offset: u8 = 16;
```
Comment (MegaRedHand) [inline tests/operations.rs:149]:
```suggestion
    let offset: u8 = 16;
    value[offset as usize] = desired_byte;
    let value: BigUint = BigUint::from_bytes_be(&value);
```

File: src/opcodes.rs (hunk 1 above)
Comment (ricomateo) [reply src/opcodes.rs]: Done!

File: src/program.rs (hunk 2 above)
Comment (ricomateo) [reply src/program.rs:30]: Done!

**Review by MegaRedHand — APPROVED** [review]
LGTM

**Pull request merged**